README.md

```diff
@@ -50,101 +53,106 @@ git clone https://github.com/Altinn/altinn-studio
 cd altinn-studio
 ```
 
-#### Altinn Studio
-
-Follow the instructions found [here](/src/studio/).
+#### Develop [Altinn Studio](/src/studio/README.md)
```

Comment:
Links as part of headers are not very readable, and may cause confusion for ToCs. Should be removed since the same link is just below.